### PR TITLE
correcting wrong assumptions of toml-files

### PR DESCRIPTION
**Pull request by ineiti**
Fixes a bug in the shell-parsing of toml-files

### CODE REVIEW

File: pop/app.go [hunk 1]
```diff
@@ -653,6 +653,7 @@ type shortDescGroupToml struct {
 // decode config of several groups into array of rosters
 func decodeGroups(buf string) ([]*service.ShortDesc, error) {
 	decodedGroups := make(map[string][]shortDescGroupToml)
+	log.Print(buf)
```
Comment (jeffallen) [inline pop/app.go]: Is this left in on purpose?

File: pop/test.sh [hunk 2]
```diff
@@ -411,10 +411,10 @@ EOF
 Location = "Earth, City$n"
 EOF
 		echo "[[parties.servers]]" >> pop_merge.toml
-		sed -n "$((5*$n-4)),$((5*$n))p" public.toml >> pop_merge.toml
+		sed -n "$((5*$n-3)),$((5*$n))p" public.toml >> pop_merge.toml
```
Comment (jeffallen) [inline pop/test.sh]: These gave me incredible difficulty the last time I had to debug them. How about the next time someone has to touch them, they find a better solution than these fiddly undocumented integres? I promise if it's me I'll do it!

Comment (ineiti) [reply pop/test.sh]: I think the integers are OK - I would care more about having a way to correctly cut up the `public.toml` into different servers and then merge them. In fact, we _do_ have that...

Comment (ineiti) [reply pop/test.sh]: OK, added it to the code.

**Pull request merged**